lib/membrane_live/event_service.ex

```diff
@@ -0,0 +1,208 @@
+defmodule MembraneLive.EventService do
```

Comment:
My concern is that we have one process for handling every join/leave action for every possible event which might be a bottle-neck in the future. What comes to my mind is to create process per event and rename this module to `Event`? Any thoughts?

Reply:
As I said in the main PR comment, I think this is a great idea but I don't want to do it in this PR.